pkg/cmd/build/build.go

```diff
@@ -194,3 +196,32 @@ func translateDockerErr(err error) error {
 	}
 	return err
 }
+
+func OptsFromManifest(name string, b *model.BuildInfo) BuildOptions {
+	if okteto.Context().IsOkteto && b.Image == "" {
+		b.Image = setOktetoImageTag(name)
+	}
+
+	opts := BuildOptions{
+		CacheFrom: b.CacheFrom,
+		Target:    b.Target,
+		Path:      b.Context,
+		Tag:       b.Image,
+		File:      filepath.Join(b.Context, b.Dockerfile),
+	}
+	if len(b.Args) != 0 {
+		opts.BuildArgs = model.SerializeBuildArgs(b.Args)
+	}
+
+	opts.OutputMode = setOutputMode("tty")
```

Comment:
we should get this fron the build flag `--progress`. There is also a `BUILDKIT_PROGRESS` envvar to configure it

Reply:
ok, this function is already using BUILDKIT_PROGRESS, and using tty as default if this is not defined, i will add the flag